finnhub/python/publisher.py

```diff
@@ -1,12 +1,11 @@
 import asyncio
 import json
-import os
 
 import websockets
 
 from pyensign.events import Event
 from pyensign.ensign import Ensign
-
+from config import CLIENT_ID,CLIENT_SECRET,FINNHUB_API_KEY
```

Comment:
Will this be an error if the `config.py` file doesn't exist? Was there a problem loading the credentials from the environment?